src/Components/WebView/WebView/src/PageContext.cs

```diff
@@ -45,10 +46,10 @@ internal class PageContext : IDisposable
             Renderer = new WebViewRenderer(services, dispatcher, ipcSender, loggerFactory, JSRuntime.ElementReferenceContext);
         }
 
-        public void Dispose()
+        public async ValueTask DisposeAsync()
         {
             Renderer.Dispose();
-            _serviceScope.Dispose();
+            await _serviceScope.DisposeAsync();
```

Comment:
Not sure how much this is called, but you could just return this if you don't mind losing the stack frame in any possible exceptions.

Reply:
Will do.